scale/storage/test/brokers/test_nfs_broker.py

```diff
@@ -107,7 +107,7 @@ def test_successfully(self):
         volume = broker.volume
         self.assertEqual(volume.driver, 'nfs')
         self.assertEqual(volume.host, False)
-        self.assertEqual(volume.remote_path, 'host/dir')  # No : character in path for Docker NFS volume
+        self.assertEqual(volume.remote_path, 'host:/dir')
```

Comment:
Comment not valid anymore why? Due to docker update?

Reply:
The Docker netshare plugin was updated and the colon substitution is no longer needed.

Reply:
👍